stdlib/public/runtime/SwiftObject.h

```diff
@@ -58,6 +58,9 @@ SWIFT_RUNTIME_EXPORT @interface SwiftObject<NSObject> {
 - (BOOL)conformsToProtocol:(Protocol *)aProtocol;
 
 - (BOOL)respondsToSelector:(SEL)aSelector;
++ (BOOL)instancesRespondToSelector:(SEL)aSelector;
```

Comment:
+instancesRespondToSelector: was already implemented, but it was not declared here.